quickwit/quickwit-config/src/source_config/mod.rs

```diff
@@ -135,15 +136,15 @@ impl SourceConfig {
     /// Returns `desired_num_pipelines` if it's a Kafka source, else 1.
```

Comment:
Nitpick, could do with updating this doc

Reply:
nice catch